src/docs/product/session-replay/performance-overhead.mdx

```diff
@@ -24,9 +24,26 @@ Sentry's Session Replay SDK takes several measures to avoid negatively impacting
 - Sentry’s event ingestion infrastructure uses [distributed Points-of-Presence (PoPs)](https://blog.sentry.io/2022/03/31/sentry-points-of-presence-how-we-built-a-distributed-ingestion/) which place ingestion servers around the world and close to your users. When a Session Replay event is transmitted, the user’s browser connects and transmits the event payload to the closest PoP available for their region. This greatly reduces end-to-end latency and minimizes the amount of networking overhead placed on the browser.
 - The SDK is built to gracefully downgrade when needed. In the event several thousand DOM mutations happen at once, the SDK will disable recording to make sure recording the website doesn't further degrade user experience. The [mutation limit values are configurable](/platforms/javascript/session-replay/configuration/#mutation-limits).
 
-## Performance Overhead in Sentry
+## Factors in Performance Overhead
 
-While the performance overhead of our client SDK will be imperceptible to end-users most of the time, it can vary based on the complexity of your application. If an applications has a large DOM and numerous DOM node mutations, it will have a higher overhead than simpler, mostly static sites. The only way to get accurate metrics is to measure performance overhead yourself.
+While the performance overhead of our client SDK will be imperceptible to end-users most of the time, it can vary based on the complexity of your application. If an applications has a large DOM and numerous DOM node mutations, it will have a higher overhead than simpler, mostly static sites.
+
+There are different stages in the lifecycle of a webpage, each with different performance characteristics. Here is a list of stages in the lifecycle, and the most important performance metrics to measure.
+
+1. Pageload
+   - [SDK bundle size is something we optimize for](https://blog.sentry.io/sentry-bundle-size-how-we-reduced-replay-sdk-by-35/)
+   - The size of your initial html page will affect how long the initial snapshot that the SDK takes, and has to send over the wire. Using your browser devtools, filter the Network panel to show "type=document" and look at the size to get a sense of the impact on your site. For example, if you show a loading screen on a single-page-app then the overhead is negligable.
```

Comment:
```suggestion
   - The size of your initial HTML page will affect how long the initial snapshot that the SDK takes, and has to send over the wire. Using your browser devtools, filter the Network panel to show "type=document" and look at the size to get a sense of the impact on your site. For example, if you show a loading screen on a single-page app, then the overhead is negligible.
```

Reply:
```suggestion
   - The size of your initial HTML page affects how long the SDK takes to complete the initial snapshot and send it over the wire. Using your browser devtools, filter the Network panel to show "type=document" and look at the size to get a sense of the impact on your site. For example, if you show a loading screen on a single-page-app then the overhead is negligible.
```
It took a while to parse the first sentence, so I adjusted the wording. Hope it still reads accurately!